src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.Library.csproj

```diff
@@ -48,6 +50,28 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <Target Name="PublishAndMergePackNupkg" AfterTargets="AfterBuild" Condition="'$(TargetFramework)' != '' AND '$(IsXPlat)' == 'false' AND '$(BuildingInsideVisualStudio)' != 'true'">
+    <MSBuild
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="Publish;ILMergeNuGetPack"/>
+  </Target>
+
+  <Target Name="ILMergeNuGetPack"  Condition="'$(TargetFramework)' != ''">
+    <PropertyGroup>
+      <ILMergeResultDir>$(OutputPath)ilmerge</ILMergeResultDir>
```

Comment:
Should this have the target framework included in the path since two different frameworks are needed for the nupkg?